### ISSUE TITLE
pretrain on source domain

### ISSUE THREAD

**Issue by wh-isper** [TFboys-lzz/MPSCL#7]
Dear author, I have a question. If i want to run train_UDA.py to train MPSCL, I need a pretrained model on source domain which you have provided, but you did not provide the code to pretrain model  on source data?

**Comment by wh-isper** [comment]
If I want to train on my own dataset from scratch, should I write a code to pretrain on source domain or just run train_UDA.py to get pretrained model and run train_UDA again to get final model?

**Comment by TFboys-lzz** [comment]
I suggest you rewrite the code to pre-train only in the source domain so as to generate the corresponding pseudo-labels for the target domain data.

**Comment by wh-isper** [comment]
I got it!
